pkg/controller/networkpolicy/clusternetworkpolicy.go

```diff
@@ -146,9 +146,10 @@ func (n *NetworkPolicyController) processClusterNetworkPolicy(cnp *secv1alpha1.C
 			appliedToGroupNamesForRule = append(appliedToGroupNamesForRule, atGroup)
 			appliedToGroupNamesSet.Insert(atGroup)
 		}
+		iFromPeers := n.toAntreaPeerForCRD(ingressRule.From, cnp, controlplane.DirectionIn, namedPortExists)
```

Comment:
“i" and "From" is a bit duplicate, and should it be singular given the method name?
I mean "fromPeer" or just "from"?